ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -389,18 +404,47 @@ func (h *FaultHandler) StartNetworkPacketLoss() func(http.ResponseWriter, *http.
 		rwMu.Lock()
 		defer rwMu.Unlock()
 
-		// TODO: Check status of current fault injection
```

Comment:
Probably a good idea to still check if the there's an existing fault before trying to start a new one. Can be done either in `StartNetworkPacketLoss` or `startNetworkPacketLossFault`. WDYT?

Reply:
If so, could we leave this comment here and remove it once the check fault status implementation is ready?

Reply:
never mind, seems like we already are. disregard!